mt-kit/core/css/src/_theme.scss

```
@@ -11,6 +12,7 @@ $focus-box-shadow-on-white: 0 0 0 0.125rem var(--color-mt-white),
 $focus-box-shadow-no-outline: 0 0 0 0.25rem var(--mt-button-primary-bg-focus);
 $error-box-shadow: 0 0 0 0.5rem var(--color-mt-background), 0 0 0 0.7rem var(--color-error);
 $error-box-shadow-small: 0 0 0 0.05rem var(--color-mt-background), 0 0 0 0.2rem var(--color-error);
+$error-box-shadow-medium: 0 0 0 1.05rem var(--color-mt-background), 0 0 0 1.2rem var(--color-error);
```

```suggestion
$error-box-shadow-medium: 0 0 0 var(--spacer-xx-small) var(--color-mt-background),
  0 0 0 calc(var(--spacer-xx-small) + $border-width-large) var(--color-error);
```
Suggest using variables for this

---

@lha047  I dont think using the same values will work, as the borders are overlapping. The error border must be bigger to show underneath the "masking" border.

Making the second border  `--spacer-x-small` will make it look too big imo

---

Thats why you calcualte the red border

---

Haha, my tired eyes just didnt see it! Sorry